### PR TITLE
Doctor Moffson Antivirus for Artificial Intelligence Units @ the AI restore consoles

### PR DESCRIPTION
**Pull request by nevimer**
## About The Pull Request

This PR makes it so that you can fix your malfunctional (naughty) AI instead of wiping and RR'ing them.
Yes, to get to this point you need to defeat the AI. Keep the player in the round, everyone benefits, etc.
https://cdn.discordapp.com/attachments/610319137012580383/1049887877972099083/dreamseeker_QRntph4a7Z.mp4
## How This Contributes To The Skyrat Roleplay Experience

Lets players do more without the fear of sitting in ghost and getting jaded so fast by the inevitable "malf bad" gang busting your doors down
SftFpIRyQm

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  

https://user-images.githubusercontent.com/77420409/206078823-826e331b-8242-49e5-abfa-ad774ea8fd75.mp4

https://user-images.githubusercontent.com/77420409/206079197-d8e38bed-609c-467f-b6e0-4776f55be4c2.mp4

</details>

## Changelog

🆑 
add: You can now un-malf your malf-AI!
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/malf/remove_malf.dm [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		restoring = TRUE
+		run_antivirus()
+		. = TRUE
+
+/datum/computer_file/program/ai_restorer/proc/run_antivirus()
+	var/mob/living/silicon/ai/bad_ai = stored_card.AI
+	while(bad_ai && restoring)
+		sleep(2 SECONDS)
+		if(prob(25))
+			bad_ai.adjustOxyLoss(rand(-5, 15), FALSE)
```
Comment (Cyprex) [inline modular_skyrat/master_files/code/modules/malf/remove_malf.dm:68]: Won't this deal damage on average?

**Comment by KathrinBailey** [comment]
Wasn't this something merged on TG/Skyrat?

**Comment by nevimer** [comment]
> Wasn't this something merged on TG/Skyrat?

It was left to stale out if I recall correctly

**Comment by StrangeWeirdKitten** [comment]
@nevimer I want to merge this. This PR is broken.

**Comment by Majkl-J** [comment]
Stale. Reopen when work on this begins again

**Pull request merged**